build/appveyor/before_build.bat

```diff
@@ -7,8 +7,8 @@ SET OLD_PATH=%PATH%
 :: copy dependencies
 START " " /wait "7z" x -y dependencies.7z > nul
 CD dependencies
-XCOPY i686-w64-mingw32 "C:\Qt\Tools\mingw491_32\i686-w64-mingw32" /E /Y
-XCOPY lib "C:\Qt\Tools\mingw491_32\lib" /E /Y
+XCOPY i686-w64-mingw32 "C:\Qt\Tools\mingw530_32\i686-w64-mingw32" /E /Y
+XCOPY lib "C:\Qt\Tools\mingw53_32\lib" /E /Y

```

Comment:
OK, again, here we need `XCOPY lib "C:\Qt\Tools\mingw530_32\lib" /E /Y`

Reply:
Aaahh! My bad. Sorted. :D

Reply:
Adding a `/I` to the `XCOPY` commands should prevent the hang